lib/install/install.go

```diff
@@ -180,9 +191,24 @@ func (i *Installer) startExecuteLoop() {
 		for {
 			select {
 			case req := <-i.execC:
-				if err := i.execute(req); err != nil {
-					i.sendError(err)
+				status, err := i.execute(req)
+				select {
+				case <-i.execDoneC:
```

Comment:
I'm not sure if I'm missing something, but why are execResults with errors being written to this channel below, if the errors get discarded if there are more requests?